students/BrianNisonger/session9/test_mailroom.py

```diff
@@ -0,0 +1,181 @@
+from mailroom import Donor
+from mailroom import Donors
+import statistics
+
+
+def test_make_donor():
+    donor = Donor("Fred Flinstone")
+    assert donor.name == "Fred Flinstone"
+
+
+def test_make_donor_with_donation():
+    donor = Donor("Fred Flinstone", [100, 200, 300, 400])
+    assert donor.name == "Fred Flinstone"
+    assert donor.donation == [100, 200, 300, 400]
```

Comment:
careful with your naming -- use plural for lists of things -- donations -- and singular for just one -- donation.